scripts/sizeSnapshot/webpack.config.js

```
@@ -97,8 +97,9 @@ async function getSizeLimitBundles() {
 
 module.exports = getSizeLimitBundles;
 
-module.exports = new Promise(async resolve => {
-  const entry = (await getSizeLimitBundles()).reduce((acc, bundle) => {
+module.exports = async function webpackConfig() {
```

Should've read the text instead of only the headline :laughing: 
https://webpack.js.org/configuration/configuration-types/#exporting-a-promise

---

Throw an exception inside the method, it's swallowed.

---

Well yes. That's the nature of unhandledPromiseRejections. Which is why I always do asyncFunction().catch()

---

It's not enough.